### PR TITLE
Added description to improved bin/console debug:container --env-vars …

### PR DESCRIPTION
**Pull request by mw3e21a**

### CODE REVIEW

File: configuration.rst [hunk 1]
```diff
@@ -871,19 +871,21 @@ Use the ``debug:dotenv`` command to understand how Symfony parses the different
...
-     BAR              null              n/a
-    ---------------- ----------------- ---------------------------------------------
+ ------------ ----------------- ------------------------------------ -------------
+  Name         Default value     Real value                           Usage count
+ ------------ ----------------- ------------------------------------ -------------
+  APP_SECRET   n/a               "471a62e2d601a8952deb186e44186cb3"   2
+  BAR          n/a               n/a                                  1
+  BAZ          n/a               "value"                              n/a
+  FOO          "[1, "2.5", 3]"   n/a                                  1
+ ------------ ----------------- ------------------------------------ -------------
```
Comment (OskarStark) [inline configuration.rst]: Please use the same indentation as before, thanks

File: configuration.rst [hunk 2]
```diff
@@ -871,19 +871,21 @@ Use the ``debug:dotenv`` command to understand how Symfony parses the different
     The option to pass variable names to ``debug:dotenv`` was introduced in Symfony 6.2.
 
 Additionally, and regardless of how you set environment variables, you can see all
-environment variables, with their values, referenced in Symfony's container configuration:
+environment variables, with their values, referenced in Symfony's container configuration, you can also
+see the number of occurrences of each environment variable in the container:
```
Comment (OskarStark) [inline configuration.rst]:
```suggestion
environment variables, with their values, referenced in Symfony's container configuration,
you can also see the number of occurrences of each environment variable in the container:
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by mw3e21a** [comment]
@OskarStark Thanks for the suggestions. Done

**Review by alamirault — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Michael, thanks and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**